app/Http/Controllers/HomeController.php

```diff
@@ -17,12 +23,98 @@ public function __construct()
     }
 
     /**
-     * Show the application dashboard.
+     * Show the application dashboard and all user posts.
      *
      * @return \Illuminate\Http\Response
      */
     public function index()
     {
-        return view('home');
+        //Get user id
+        $user_id = Auth::user()->id;
+
+        //Find all user's posts
+        $posts = Post::where('user_id', $user_id)->get();
```

Comment:
```suggestion
      //Find all user's posts
        $posts = Post::where('user_id', Auth::user()->id)->get();
```

Reply:
Makes sense as I don't use the variable again.